src/python/WMCore/WMSpec/WMTask.py

```diff
@@ -1349,6 +1349,13 @@ def setAcquisitionEra(self, era, parentAcquisitionEra=None):
         """
         if isinstance(era, dict):
             taskEra = era.get(self.name(), parentAcquisitionEra)
+            if taskEra is None:
+                # We cannot properly set AcqEra for ACDC of TaskChain Merge
+                # failures, so we should look up for a similar taskname in
+                # the acqera dict passed from the requestor
+                for taskname in era:
```

Comment:
Hmm, I had that in mind as well. But in this specific case, I don't think it has a parent task, since the Merge task would be the first Task in the ACDC request.